hasher-matcher-actioner/scripts/set_threatexchange_source

```diff
@@ -34,7 +34,7 @@ elif [ $1 == "local" ]; then
 
     # Copy and install from source. Copying is necessary. A link would not work.
     # Dockerbuilds can't follow symlinks outside their context directory.
-    rm -r $LOCAL_LIB_LOCATION 2>/dev/null
+    rm -r $LOCAL_LIB_LOCATION 2>/dev/null || true 
```

Comment:
Is there an rm flag you can use or gate this instead with an if? Otherwise you permission fail your removal this may behave unexpectedly.

Also feeds into my opinion of "use python for scripts if script is longer than a few lines"